doc/man3/OCSP_sendreq_new.pod

```diff
@@ -126,7 +130,19 @@ L<OCSP_response_status(3)>
 
 =head1 HISTORY
 
-The OCSP_REQ_CTX_set1_req() function was deprecated in OpenSSL 3.0.
+The OCSP-specific names for types and functions were deprecated in OpenSSL 3.0.
+
+The B<OCSP_REQ_CTX> typedef is an alias for B<OSSL_HTTP_REQ_CTX>.
+
+The OCSP_REQ_CTX_set1_req(), OCSP_REQ_CTX_free(),
+OCSP_REQ_CTX_add1_header(),
+OCSP_REQ_CTX_set1_req(),
+OCSP_set_max_response_length()
```

Comment:
In this deprecation list I am missing
OCSP_REQ_CTX_new
OCSP_REQ_CTX_http
OCSP_REQ_CTX_i2d
OCSP_REQ_CTX_nbio
OCSP_REQ_CTX_nbio_d2i
OCSP_REQ_CTX_get0_mem_bio

Reply:
> In this deprecation list I am missing

I added those that were documented; I did not mark undocumented functions as deprecated. I also put them in the same ordering as the fixed synopsis/text sections.

Reply:
> I added those that were documented; I did not mark undocumented functions as deprecated.

All right, but `OCSP_REQ_CTX_i2d` is documented.

> I also put them in the same ordering as the fixed synopsis/text sections.

Looks like you forgot to push your changes.

Reply:
> Looks like you forgot to push your changes.

i blame Santa.